### DISCUSSION TITLE
Flows - calling secure endpoint via webhook never seems to work

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#13859] [Archive]
Hey everyone, 

I have been having an issue calling any third-party API endpoint that requires an authentication header to be passed. 

Not sure what's causing it; calling the same call via postman works fine. However, invoking the same call via Directus doesn't work. 

I am attaching the "log to console" step after Webhook doesn't reveal any info. 

![image](https://user-images.githubusercontent.com/1252959/173273324-7f23b3b3-95eb-4aaf-8144-527ff5d67d91.png)

Can someone help me verify if this is a bug? 

Thanks

**Comment by azrikahar** [comment]
What Directus version are you on? This sounds similar to #13737 reported in `9.12.1`, but it should be resolved in `9.12.2`.

**Reply by DeanKamali** [reply]
Thanks @azrikahar  upgrade fixed it.
